### PR TITLE
Switch to ruby version of connect vbms

### LINKED ISSUE #309: Migrate to Ruby-only Connect VBMS

### PR DESCRIPTION
**Pull request by shanear**
fixes #309

### CODE REVIEW

**Comment by orischwartz** [comment]
For reference: https://github.com/department-of-veterans-affairs/caseflow-efolder/pull/114

Above PR includes the required Travis changes plus some other notes to make sure we don't forget anything. (Unrelated, looks like we never removed `connect_vbms` from eFolder Express's `vendor` dir).

File: app/services/appeal_repository.rb [hunk 1]
```diff
@@ -121,17 +121,6 @@ def self.init_vbms_client
     return VBMS::Client.from_env_vars(
       logger: CaseflowLogger.new,
       env_name: ENV["CONNECT_VBMS_ENV"]
-    ) if Rails.application.secrets.vbms["env"]
```
Comment (orischwartz) [inline app/services/appeal_repository.rb]: Any reason we're removing this? It's still used this way in [efolder](https://github.com/department-of-veterans-affairs/caseflow-efolder/blob/master/app/services/vbms_service.rb#L37).

**Comment by shanear** [comment]
@orischwartz travis is showing ruby 2.3.0, we should probably upgrade ruby to 2.3.0 for certification too.

**Comment by orischwartz** [comment]
@shanear, got it looks like I misread the Travis output. There's just one small rubocop issue here:

```
app/services/appeal_repository.rb:121:5: C: Redundant return detected.
    return VBMS::Client.from_env_vars(
    ^^^^^^
69 files inspected, 1 offense detected
```

**Comment by shanear** [comment]
@orischwartz fixed

**Comment by orischwartz** [comment]
To be merged simultaneously with https://github.com/department-of-veterans-affairs/appeals-deployment/pull/152

**Pull request merged**